### PR TITLE
Update changelog and add changelog-checker CI 

### LINKED ISSUE #139: Keep the changelog up to date
I noticed that I opened several PR without updating the changelog. For instance #61 #90. I was thinking to open a PR where I update the changelog and I also add this simple action to remind the author of the PR to update the changelog https://github.com/marketplace/actions/changelog-checker  

As far as I understood the setting `noChangelogLabel` can be used to avoid running the CI.

@traversaro @S-Dafarra @prashanthr05 do you have any suggestion?

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
This should fix #139

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,14 @@ All notable changes to this project are documented in this file.
 - Implement `ParametersHandler` library (https://github.com/dic-iit/bipedal-locomotion-controllers/pull/13)
 - Implement `GenericContainer::Vector` (https://github.com/dic-iit/bipedal-locomotion-controllers/pull/29)
 - Implement `Estimators` library (https://github.com/dic-iit/bipedal-locomotion-controllers/pull/23)
-- Renamed from ``bipedal-locomotion-controllers`` to ``bipedal-locomotion-framework`` (https://github.com/dic-iit/bipedal-locomotion-framework/pull/40).
+- Rename from `bipedal-locomotion-controllers` to `bipedal-locomotion-framework` (https://github.com/dic-iit/bipedal-locomotion-framework/pull/40).
```
Comment (traversaro) [inline CHANGELOG.md:17]: Not sure it make sense to have this line. The CHANGELOG is meant to track changes from one release to another, as this is the first changelog of an unreleased library, probably we can omit this.

**Review by traversaro — APPROVED** [review]
Nitpick comment, but it can be approved anyhow.

File: CHANGELOG.md (hunk 1 above)
Comment (GiulioRomualdi) [reply CHANGELOG.md:17]: Ok So I can remove that line

Comment (GiulioRomualdi) [reply CHANGELOG.md]: Done in 6f43ec8776e5ea89c460d471bc0410852bc6c9f5

**Comment by prashanthr05** [comment]
Shall I add changelog updates relevant to `FloatingBaseEstimators` as within the scope of this PR?

**Review by S-Dafarra — APPROVED** [review]

**Comment by GiulioRomualdi** [comment]
If you agree we can merge the PR. Then if in the future we understand that the new action is a pain in the neck we remove it

**Comment by traversaro** [comment]
> 
> 
> If you agree we can merge the PR. Then if in the future we understand that the new action is a pain in the neck we remove it

Ok for me!

**Pull request merged**